.drone.yml

```diff
@@ -85,29 +85,63 @@ steps:
 
   - name: integration
     image: enigmampc/docker-client
+    privileged: true
     depends_on:
       - core
       - principal
+    volumes:
+      - name: sock
+        path: /var/run/docker.sock
```

Comment:
why is necessary? what changed that it is needed in the integration tests as well?

Reply:
Nothing has changed, I am only re-ordering lines. If you take a look at the file before these changes, these lines were already present in 105-107. Because the container needs to access docker on the host, it needs access to the socker.